src/main/java/nl/knaw/dans/migration/core/DataverseLoader.java

```diff
@@ -83,14 +97,12 @@ public void loadFromDataset(String doi) {
                 ++fileCount;
             }
             log.info("Stored {} actual files for DOI {}, Version {}.{} State {}", fileCount, doi, v.getVersionNumber(), v.getVersionMinorNumber(), v.getVersionState());
-            List<MetadataField> citation = v.getMetadataBlocks().get("citation").getFields();
-            PrimitiveSingleValueField depositor = (PrimitiveSingleValueField) citation.stream().filter(f -> "depositor".equals(f.getTypeName())).findFirst().orElse(null);
             ActualDataset actualDataset = new ActualDataset();
             actualDataset.setMajorVersionNr(v.getVersionNumber());
             actualDataset.setMinorVersionNr(v.getVersionMinorNumber());
             actualDataset.setDoi(shortDoi);
-            actualDataset.setDepositor(depositor.getValue());
-            actualDataset.setAccessCategory(null);
+            actualDataset.setDepositor(depositor);
+            actualDataset.setAccessCategory("FileAccessRequest="+v.isFileAccessRequest());// TODO change table field?
```

Comment:
the name of this property is a bit misleading. It is about whether restricted files can be requested, right?

Reply:
indeed, the TODO is about that, I'm renaming it in the next PR